go/libkb/chain_link.go

```diff
@@ -286,7 +285,11 @@ func (c *ChainLink) GetUID() keybase1.UID {
 }
 
 func (c *ChainLink) GetPayloadJSON() *jsonw.Wrapper {
-	return c.payloadJSON
+	payloadJSON, err := jsonw.Unmarshal([]byte(c.unpacked.payloadJSONStr))
+	if err != nil {
+		return nil
```

Comment:
Urg, this is pretty scary, we're swallowing an error here....

Reply:
It would be nice to surface this error somehow...

Reply:
The error will be surfaced in Unpack().